apps/web/src/components/Feed/FeedList.tsx

```diff
@@ -171,8 +171,10 @@ export default function FeedList({
       {({ height, scrollTop, registerChild }) => (
         <AutoSizer disableHeight>
           {({ width }) => (
+            // calling registerChild resets the scroll position which fixes an issue where upon remounting this list, virtualization didn't work
+            // https://github.com/bvaughn/react-virtualized/issues/1324
             // @ts-expect-error shitty react-virtualized types
-            <div ref={registerChild}>
+            <div ref={(el) => registerChild(el)}>
```

Comment:
acknowledging this means we're defining this function on every render. fortunately this component doesn't get re-rendered frequently - only when loading more data. so the upside of fixing a major issue on the feed seems worth this cost.